.env.example

```diff
@@ -0,0 +1 @@
+API_URL="localhost:8000"
```

Comment:
Change the env var name to `Templates_API_URL` and port to `8080`. Also, add these two env vars:
- `Checkups_API_URL` with port `8081`
- `User_API_URL` with port `8082`

Reply:
Done 👍